cas-server-support-saml-idp/build.gradle

```diff
@@ -0,0 +1,143 @@
+description = 'Apereo CAS SAML Identity Provider Support'
+dependencies {
+    compile project(':cas-server-core-services')
+    compile(project(':cas-server-support-saml')) {
+        exclude(module: 'xml-apis')
+        exclude(module: 'guava')
+    }
+
+    compile group: 'org.apache.commons', name: 'commons-lang3', version:commonsLangVersion
+    compile(group: 'org.jasig.cas.client', name: 'cas-client-core', version: casClientVersion) {
+        exclude(module: 'servlet-api')
+        exclude(module: 'log4j-over-slf4j')
+        exclude(module: 'bcprov-jdk15')
+        exclude(module: 'slf4j-api')
+    }
+    compile(group: 'net.shibboleth.idp', name: 'idp-core', version: idpVersion) {
```

Comment:
Definitely verbose all these `exclude`: we loose the Maven transitivity here, but definitely less a nightmare for CAS deployers who don't need to exclude dependencies everywhere: I (used to) do that in my overlays.